### DISCUSSION TITLE
[help] Interpolate heigth from a low-poly mesh to a high-poly one

### DISCUSSION THREAD

**Discussion by raphael2692** [cnr-isti-vclab/meshlab#819] [Q&A]
Hello, 

I have the following problem and I would love to know if anyone can help me solving it with meshlab. I have a low-poly mesh obtained from photogrammetry and it is aligned and scaled to a high-poly mesh representing the same object. I need to create an heatmap representig the quotas (z-ccords) over the high poly model. I was able to achieve something similar via the Hausdorff Distance filter, however that metric doesn't fit my needs. Is there a way to achieve this in meshlab?

Thanks!

**Comment by jmespadero** [comment]
Have you tried the `distance from reference mesh` filter and transform quality values (distances) to colors using the quality mapper? 

Also, could you paste one screenshot to illustrate your problem and see what's the problem with Hausdorff distance?
